background_scripts/main.js

```diff
@@ -350,6 +360,58 @@ const BackgroundCommands = {
   }
 };
 
+chrome.commands.onCommand.addListener(function(command) {
+
+  const sendCommandToCurrentTab = function(requestName) {
+    chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+      if (tabs[0]) {
+        const tabId = tabs[0].id;
+        chrome.tabs.sendMessage(tabId, {
+          name: 'runInTopFrame',
+          registryEntry: {
+            command: requestName,
+            optionList: [],
+          },
+        });
+      }
+    });
+  };
+
+  switch (command) {
+    case "jump-back-tab":
+      BackgroundCommands.jumpBackTabList({count: 1});
+      break;
+    case "jump-forward-tab":
+      BackgroundCommands.jumpForwardTabList({count: 1});
+      break;
+    case "open-vomnibox":
+      sendCommandToCurrentTab("Vomnibar.activate");
+      break;
+    case "open-vomnibox-new-tab":
+      sendCommandToCurrentTab("Vomnibar.activateInNewTab");
+      break;
+    case "open-vomnibox-tab":
+      sendCommandToCurrentTab("Vomnibar.activateTabSelection");
+      break;
+    case "open-vomnibox-bookmark":
+      sendCommandToCurrentTab("Vomnibar.activateBookmarks");
+      break;
+    case "open-vomnibox-bookmark-new-tab":
+      sendCommandToCurrentTab("Vomnibar.activateBookmarksInNewTab");
+      break;
+    case "switch-to-previous-tab":
+      chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+        if (tabs[0]) {
+          const tabId = tabs[0].id;
+          BackgroundCommands.visitPreviousTab({count: 1, tab: { id: tabId} });
+        }
+      });
+      break;
+    default:
+      console.error('unrecognized command: ', command);
```

Comment:
You don't need to add a space after the string. The logger joins multiple things together with spaces by default.